File: IsometricShooter/Scripts/JS/Configuration/ObstacleConf.js
#pragma strict

var onHitEffect : GameObject; //Effect on hit by projectile
var onHitEffectType : Effect; //Type of effect on hit by projectile
var onDieEffect : GameObject; //Effect on die by projectile
var onDieEffectType : Effect; //Type of effect on die by projectile
var onDieCameraAnimation : AnimationClip;
var onDieArealDamage : int = 0; //Areal damage in case of impact radius > 0
var onDieImpactRadius : float = 0.0; //Damage radius
var onDieEffectWaitSeconds : float = 0.0;
var sfxHit : AudioClip;
var sfxDie : AudioClip;
var getXpOnKill : int = 10; //XP the player gains on kill of this NPC
var getXpColor : Color = Color.yellow;
var killLoot : GameObject; //Object spawned as kill loot ("None" = nothing will spawn)

File: IsometricShooter/Scripts/CS/Configuration/ObstacleConf.cs
using UnityEngine;
using System.Collections;

public class ObstacleConf : MonoBehaviour {

	public GameObject onHitEffect; //Effect on hit by projectile
	public Effect onHitEffectType; //Type of effect on hit by projectile
	public GameObject onDieEffect; //Effect on die by projectile
	public Effect onDieEffectType; //Type of effect on die by projectile
	public AnimationClip onDieCameraAnimation;
	public int onDieArealDamage = 0; //Areal damage in case of impact radius > 0
	public float onDieImpactRadius = 0.0f; //Damage radius
	public float onDieEffectWaitSeconds = 0.0f;
	public AudioClip sfxHit;
	public AudioClip sfxDie;
	public int getXpOnKill = 10; //XP the player gains on kill of this NPC
	public Color getXpColor = Color.yellow;
	public GameObject killLoot; //Object spawned as kill loot ("None" = nothing will spawn)
	
}
